source/extensions/common/aws/signer_base_impl.h

```diff
@@ -112,6 +116,7 @@ class SignerImpl : public Signer, public Logger::Loggable<Logger::Id::aws> {
   TimeSource& time_source_;
   DateFormatter long_date_formatter_;
   DateFormatter short_date_formatter_;
+  const std::string BlankStr = std::string();
```

Comment:
```
const std::string blank_str_;
```

Style, and the `=std::string()` is meaningless.

Reply:
ack